.devcontainer/prometheus/prometheus.yml

```diff
@@ -0,0 +1,24 @@
+global:
+  scrape_interval: 15s # By default, scrape targets every 15 seconds.
+
+scrape_configs:
+  - job_name: "psinode_logs"
+    scrape_interval: 1s
+    static_configs:
+      - targets: ["localhost:9144"]
+
+  - job_name: "psinode_perf"
+    scrape_interval: 1s
+    metrics_path: /native/admin/metrics
+    static_configs:
+      - targets: ["host.docker.internal:8080"]

```

Comment:
I thought `psinode` is running running in the container in this setup. This appears to be saying that `psinode` is running on the host.

Reply:
Fixed